src/cam_poa.h

```diff
@@ -0,0 +1,49 @@
+/*
+*  cam_poa.h
+*  PHD Guiding
+*
+*  Created by Robin Glover.
+*  Copyright (c) 2014 Robin Glover
+*  Copyright (c) 2018 Andy Galasso
+*  Copyright (c) 2024 PHD2 Developers
+*  All rights reserved.
+* 
+*  Created by Ethan Chappel based on cam_zwo.h
+*
+*  This source code is distributed under the following "BSD" license
+*  Redistribution and use in source and binary forms, with or without
+*  modification, are permitted provided that the following conditions are met:
+*    Redistributions of source code must retain the above copyright notice,
+*     this list of conditions and the following disclaimer.
+*    Redistributions in binary form must reproduce the above copyright notice,
+*     this list of conditions and the following disclaimer in the
+*     documentation and/or other materials provided with the distribution.
+*    Neither the name of Craig Stark, Stark Labs nor the names of its
```

Comment:
our licenses are a bit of a mess.  Let's use one of the most recently created files as a template, like cam_ogma.cpp: 
```suggestion
*    Neither the name of openphdguiding.org nor the names of its
```